code/modules/modular_computers/file_system/programs/generic/email_client.dm

```diff
@@ -177,22 +178,22 @@
 	last_message_count = 0
 	read_message_count = 0
 
-/datum/nano_module/email_client/ui_interact(mob/user, ui_key = "main", var/datum/nanoui/ui = null, var/force_open = 1, var/datum/topic_state/state = GLOB.default_state)
+/datum/nano_module/program/email_client/ui_interact(mob/user, ui_key = "main", var/datum/nanoui/ui = null, var/force_open = 1, var/datum/topic_state/state = GLOB.default_state)
 	var/list/data = host.initial_data()
 
 	// Password has been changed by other client connected to this email account
 	if(current_account)
 		if(current_account.password != stored_password)
 			if(!log_in())
 				log_out()
-				error = "Invalid Password"
```

Comment:
why are you changing this everywhere to program's var?

Reply:
There was copy pasting everywhere. That's why I moved the whole namespace to /program, because I made error at the program level rather than the nano module level.